tests/integration/test_content_manifest.py

```diff
@@ -39,3 +43,42 @@ def test_valid_content_manifest_request(test_env, default_requests):
 
         response_data = content_manifest_response.data
         utils.assert_content_manifest_schema(response_data)
+
+
+def test_invalid_sbom_request(test_env: Dict[str, Any]) -> None:
+    """
+    Send an invalid sbom request to the Cachito API.
+
+    Checks:
+    * Check that the response code is 400
+    """
+    client = utils.Client(test_env["api_url"], test_env["api_auth_type"], test_env.get("timeout"))
+
+    with pytest.raises(requests.HTTPError) as e:
+        client.fetch_sbom(request_ids=0)
+    assert e.value.response.status_code == 400
+    assert e.value.response.json() == {"error": "Cannot find request(s) 0."}
+
+
+def test_valid_sbom_request(test_env: Dict[str, Any], default_requests: Request) -> None:
```

Comment:
```suggestion
def test_valid_sbom_request(test_env: Dict[str, Any], default_requests: Dict[str, DefaultRequest]) -> None:
```
where DefaultRequest comes from tests/integration/conftest.py

Reply:
ok